docs/code-search/code-navigation/syntactic_code_navigation.mdx

```diff
@@ -0,0 +1,42 @@
+# Syntactic Code Navigation
+
+<TierCallout>
+  Supported on [Enterprise](/pricing/enterprise) plans.
+  <user>
+   This feature is currently experimental and enabled by default for Cloud customers
+  </user>
+</TierCallout>
+
+Syntactic code navigation is an opt-in zero configuration feature that improves code navigation for certain languages in the absence of
+[Precise code navigation](./precise_code_navigation) set up. It works by periodically indexing repositories for which it is [enabled](#enabling-syntactic-code-navigation), using high level syntax analysis heuristics.
+
```

Comment:
Consider adding a version of the image / chart you made on search vs syntactic vs precise